### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by EnzoMartin**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -106,8 +106,8 @@
     "no-shadow": 2,
     "no-undef-init": 2,
     "no-undef": 2,
-    "no-undefined": 1,
-    "no-unused-vars": 1,
+    "no-undefined": 0,
+    "no-unused-vars": 0,
```
Comment (viart) [inline .eslintrc]: But why those 2?

Comment (EnzoMartin) [inline .eslintrc]: No-one reads warnings. Either it's an error or it's ignored.

Comment (viart) [inline .eslintrc]: ah, then what about making them as errors? ;)

Comment (EnzoMartin) [inline .eslintrc]: PR it and discuss in slack

Comment (m64253) [inline .eslintrc]: Or revert this and discuss in slack :P

Comment (EnzoMartin) [inline .eslintrc]: Most of our repos spew out so many warnings that you don't even bother reading them

**Pull request merged**